### DISCUSSION TITLE
autosend client commands. additionally: chat logs

### DISCUSSION THREAD

**Discussion by BlueNotebooks** [ORelio/Minecraft-Console-Client#1910] [Q&A] [answered]
how can i have the client autosend client commands like /dropitem ? whenever i put it in a script the client sends it like a game command and not a client command.

also but separately: the standard log files dont contain messages from any of my other accounts or server messages. how can i fix this?

**Comment by ORelio** [comment, marked as answer]
Use `dropitem` not `send dropitem`.
You may need to adjust the `[ChatFormat]` section of your configuration file, see [detecting chat messages](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#detecting-chat-messages).

**Reply by BlueNotebooks** [reply]
thanks, got the dropitem to work.
im having trouble with the chatformatting though. i dont really know much at all about programming. is there an easy way to tell it to log all in-game messages?

**Reply by ORelio** [reply]
Yon can set `filter=all` in the `[ChatLog]` section in config, in that case everything is logged even if format is not properly set.

**Reply by IchiroMisumi** [reply]
@BlueNotebooks how did you get the dropitem to work? I'm trying to do the same but then with /useitem

**Reply by BlueNotebooks** [reply]
> 
> @BlueNotebooks how did you get the dropitem to work? I'm trying to do the same but then with /useitem

the way i have it working is:
in MinecraftClient.ini 
`inventoryhandling=true`
[ScriptScheduler] 
`enabled=true`

then in the tasks.ini file for the script i have:
`[Task]`
`#drop`
`triggerOnInterval=true`
`timeInterval=1`
`action=script secondcommand.txt`
and there are more things for scripts you can use. visit [sample-tasks.ini](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/config/sample-tasks.ini) if you need it

then i have a txt file named "seccondcommand.txt" and in it:
`dropitem IronBlock`

thats how i got dropitem to work
depending on what youre doing you might also need to set `terrainandmovements=true` for the action to work
